e2e/db.js

```diff
@@ -0,0 +1,40 @@
+import mongoose, { mongo } from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+
+const MONGO_MEMORY_SERVER_PORT = process.env.MONGO_MEMORY_SERVER_PORT || 10000;
+const MONGODB_URL = `mongodb://127.0.0.1:${MONGO_MEMORY_SERVER_PORT}/`;
+
+let mongoServer = null;
+let dbInstance = undefined;
+
+const connectDB = async () => {
+  try {
+    await mongoose.disconnect();
+    mongoServer = await MongoMemoryServer.create({
+      instance: {
+        port: +MONGO_MEMORY_SERVER_PORT,
```

Comment:
Can't we give default port instead of doing type coercion on possible `undefined` value? Just a suggestion.